node/caching-compiler.js

```diff
@@ -103,16 +111,18 @@ function readFileOrNull(filename) {
   }
 }
 
-function getCacheFilename(content, reify) {
-  const hash = createHash("sha1")
-    .update(reifyVersion).update("\0")
-    .update(content).update("\0");
-
-  if (reify) {
-    hash.update(JSON.stringify(reify));
+function getCacheFilename() {
+  const args = [];
+  const argc = arguments.length;
+  for (let i = 0; i < argc; ++i) {
+    args.push(arguments[i]);
   }
 
-  return hash.digest("hex") + ".js";
+  return createHash("sha1")
```

Comment:
If it helps with flow you can create the values to add to `createHash("sha1")` as an array and then do
`createHash("sha1").update(values.join("\0")).digest("hex")`